.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
```

Comment:
A line in CHANGES_NEXT_RELEASE about the removal of Node 6 should be include. For homogeneity, I'd suggest to use the same in https://github.com/telefonicaid/iotagent-json/pull/400:

> Set Nodejs 8 as minimum version in packages.json (effectively removing Nodev6 from supported versions)

Reply:
Fixed: ab10c3b